test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -498,6 +512,29 @@ private void waitForPendingRollupTasks() throws Exception {
         waitForPendingTasks(adminClient(), taskName -> taskName.startsWith("xpack/rollup/job") == false);
     }
 
+    private static void deleteAllPolicies() throws IOException {
+        Map<String, Object> policies;
+
+        try {
+            Response response = adminClient().performRequest(new Request("GET", "/_ilm"));
+            policies = entityAsMap(response);
+        } catch (ResponseException e) {
```

Comment:
this is caught here because of https://github.com/elastic/elasticsearch/pull/34881

Reply:
Maybe add a comment referencing the issue and your intention to remove this once it is fixed. That'd be nice for someone reading the code after the fact I think.